src/systems/a320_systems/src/pneumatic.rs

```diff
@@ -814,7 +814,7 @@ impl EngineBleedAirSystem {
             ),
             engine_starter_exhaust: PneumaticExhaust::new(3e-2, 3e-2, Pressure::new::<psi>(0.)),
             engine_starter_valve: DefaultValve::new_closed(),
-            precooler: Precooler::new(5.),
+            precooler: Precooler::new(180. * 2.),
```

Comment:
What does this `* 2` signify? I assume there is a reason why this is done explicitly.

Reply:
The `* 2` signifies the area of exchange and the `180.` is the heat transfer coefficient.